Solar-System

```diff
@@ -0,0 +1,97 @@
+# Brittany Jones
+# Ada Solar system
+# Last edited 2/9/18
+
+# Create a SolarSystem class with an @planets instance variable.
+# Create an initialize method which should take a collection of planet names and store them in an @planets instance variable.
+
+# Create a class with a hash of planets and their attributes.
+# Give each planet a year_length attribute which is the length of time the planet takes to go around its star.
+# Give each planet a distance_from_the_sun attribute
+
+class Planet
+  # Create reader methods to give a user access to read the instance variables.
+attr_accessor :planet_hash
```

Comment:
Code inside a class should be indented.  Either use 1 tab or 2 spaces to indent code.

Also why make this a hash or an accessor? 

By being a hash a user of `Planet` must know the structure of the hash and they keys.  

By using `attr_accessor` you are giving users the ability to **change** `@planet_hash`.  You probably don't want to do that.

Granted in this project it's a minor issue.